.ci/transifex_upload.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Be verbose, and stop with error as soon there's one
+set -ev
+
+
+pip install virtualenv
+virtualenv ~/env
+source ~/env/bin/activate
+pip install transifex-client sphinx-intl
+pip install ".[docs,testing]"
+
+# Create the main database
+TEST_AIIDA_BACKEND="django"
+PSQL_COMMAND="CREATE DATABASE $TEST_AIIDA_BACKEND ENCODING \"UTF8\" LC_COLLATE=\"en_US.UTF-8\" LC_CTYPE=\"en_US.UTF-8\" TEMPLATE=template0;"
+psql -h localhost -c "${PSQL_COMMAND}" -U postgres -w
+# Setup the main profile
+verdi setup --profile $TEST_AIIDA_BACKEND \
+    --email="aiida@localhost" --first-name=AiiDA --last-name=test --institution="AiiDA Team" --password 'secret' \
+    --db-engine 'postgresql_psycopg2' --db-backend=$TEST_AIIDA_BACKEND --db-host="localhost" --db-port=5432 \
+    --db-name="$TEST_AIIDA_BACKEND" --db-username=postgres --db-password='' \
+    --repository="/tmp/repository_${TEST_AIIDA_BACKEND}/" --non-interactive
+
+# Set the main profile as the default
+verdi profile setdefault $TEST_AIIDA_BACKEND
+
+# Set the polling interval to 0 otherwise the tests take too long
+verdi config runner.poll.interval 0
+
+# Generate pot file and upload to transifex platform
+TRANSIFEX_PROJECT_NAME="aiida-core"
+sphinx-build -b gettext docs/source locale
+tx init --no-interactive
```

Comment:
You mentioned that the API documentation should be excluded.

The way to achieve this (tested locally) is to add the following lines [here](https://github.com/aiidateam/aiida_core/blob/6cd8c6ef4ce01ccf864b0c3a60d7d4d7cb0e9d44/docs/source/conf.py#L304):
```python
 def setup(app):
-    app.connect('builder-inited', run_apidoc)
+    # Build of API documentation may be disabled
+    # (used for translation for transifex)
+    if os.environ.get('RUN_APIDOC', None) != 'False':
+        app.connect('builder-inited', run_apidoc)
```
and to `export RUN_APIDOC=False`